percona-server-8.0/Dockerfile

```diff
@@ -47,18 +46,21 @@ RUN set -ex; \
     rpm --import /etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY; \
     percona-release disable all; \
     percona-release enable ps-80 ${PS_REPO}; \
-    percona-release enable mysql-shell ${PS_REPO}
+    percona-release enable telemetry ${PS_REPO}; \
+    percona-release enable mysql-shell testing
 
 RUN set -ex; \
     dnf -y install epel-release; \
     rpm -e --nodeps tzdata; \
     dnf -y install \
         hostname \
+        wget \
```

Comment:
Can we use curl?

Reply:
I added "wget" to download something just to debug the issue and it isn't necessary for the docker image. I have deleted it now.